app/src/display/Kconfig

```diff
@@ -0,0 +1,33 @@
+
+menuconfig ZMK_DISPLAY
+    bool "Enable ZMK Display"
+    default n
+    select DISPLAY
+    select LVGL
+	select LVGL_THEMES
+	select LVGL_THEME_MONO
+
+if ZMK_DISPLAY
+
+choice LVGL_TXT_ENC
+    default LVGL_TXT_ENC_UTF8
+
+endchoice
+
+choice ZMK_DISPLAY_STATUS_SCREEN
+    prompt "Default status screen for displays"
+    default ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN
+
+config ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN
+    bool "Built in status screen"
+    select ZMK_WIDGET_BATTERY_STATUS
+    select ZMK_WIDGET_OUTPUT_STATUS
```

Comment:
Can any of these `select`s be switched to `depends on`? Although by the looks of it none of these should have issues with being selects.

Reply:
Reading the caveats here, I think there's not much risk in going w/ `select` here. If you pick this, it should be selecting those other symbols as well

Reply:
I actually refactored this to make them `default y if BT`, then made the default built in status screen handle them being enabled or not.